### ISSUE TITLE
Manually reloading Connections.plist/ConnectionGroups.plist

### ISSUE THREAD

**Issue by oorahduc** [TablePlus/TablePlus#1754]
1. Which driver are you using and version of it (Ex: PostgreSQL 10.0):
N/A

2. Which TablePlus build number are you using (the number on the welcome screen, Ex: build 81):
Version 3.1.2 (296)

3. Feature Request:
I routinely update my Connections.plist from an external API. Each time I do so, I have to ensure that I kill TablePlus prior and in the process lose all my query results.

A nice-to-have, headache-solving feature would be the ability to manually refresh the plists in TablePlus so that it does not have to be closed and reopened.

**Comment by huyphams** [comment]
Hi @oorahduc have you tried to change the connection in the build 296? From the build 296, TablePlus will watch the connection list realtime so we do not refresh it manually.

**Comment by oorahduc** [comment]
Hi @huyphams. Yes, I am using build 296. I'm using python3's plistlib library to modify the Connections.plist. I confirmed that the plist file is updated with the new username, but when opening a new connection it still attempts to use the old username. Only after restarting TablePlus will it use the new username.

**Comment by huyphams** [comment]
Thanks for confirming @oorahduc I will take a look at it.
